docs/components.md

```diff
@@ -21,28 +21,44 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-    Client         Virt API    VM TPR  Virt Controller  k8s         VM Handler
-    -------------- ----------- ------- ---------------- ----------- ----------
-
-                   listen <----------- WATCH /vms
-                   listen <---------------------------------------- WATCH /vms
-                                          |                            |
-    POST /vms ---> validate               |                            |
-                   create ---> VM ---> observe                         |
-                     |          |         v                            |
-                     |          |      POST /pods ----> validate       |
-                     |          |         |             create         |
-                     |          |         |             schedPod -> observe
-                     |          |         |                            v
-                     | <------------------------------------------> GET /vms
-                     |          |         |                         defineVM
-                     |          |         |                         watchVM
-                     |          |         |                            |
-    DELETE /vms -> validate     |         |                            |
-                   delete ----> * --------------------------------> observe
-                     |                    |                         shutdownVM
-                     |                    |                            |
-                     :                    :                            :
+```
+Client         Virt API    VM TPR  Virt Controller  k8s         VM Handler
+-------------- ----------- ------- ---------------- ----------- ----------
+
+               listen <----------- WATCH /vms
+               listen <---------------------------------------- WATCH /vms
+                                      |                            |
+POST /vms ---> validate               |                            |
+               create ---> VM ---> observe                         |
+                 |          |         v                            |
+                 |          |      POST /pods ----> validate       |
+                 |          |         |             create         |
+                 |          |         |             schedPod -> observe
+                 |          |         |                            v
+                 | <------------------------------------------> GET /vms
```

Comment:
Would it make more sense to write  "observe" instad of GET, since virt-handler does not do a GET here?